tests/src/test/java/org/eclipse/hono/tests/amqp/AdapterAmqpIT.java

```diff
@@ -64,17 +48,14 @@
 
     private final Logger LOG = LoggerFactory.getLogger(getClass());
 
-    private static final long TEST_TIMEOUT = 10000; // ms
-    private static final int MESSAGES_TO_SEND = 60;
-
     /**
      * Sets up and initializes the integration test setup.
      * 
      * @param context The Vert.x test context.
      */
     @BeforeClass
     public static void init(final TestContext context) {
-        vertx = Vertx.vertx();
+        vertx = Vertx.vertx(new VertxOptions().setBlockedThreadCheckInterval(1000 * 60 * 60));
```

Comment:
what is the purpose of this?

Reply:
sorry, i forgot to remove. i only use it when debugging